example/02/main_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+   // "testing"
+   "log"
+	"math/rand"
+	"github.com/axamon/gobrain"
```

Comment:
Shouldn't it be `github.com/goml/gobrain`?

Reply:
yea fixed it